src/scripts/mapDiseaseOutbreakToAlerts.ts

```diff
@@ -0,0 +1,251 @@
+import { AlertD2Repository } from "../data/repositories/AlertD2Repository";
+import { NotificationD2Repository } from "../data/repositories/NotificationD2Repository";
+import { OptionsD2Repository } from "../data/repositories/OptionsD2Repository";
+import { Future } from "../domain/entities/generic/Future";
+import { getUserGroupsByCode } from "../data/repositories/utils/MetadataHelper";
+import { NotifyWatchStaffUseCase } from "../domain/usecases/NotifyWatchStaffUseCase";
+import {
+    getNotificationOptionsFromTrackedEntity,
+    getOutbreakKey,
+    mapTrackedEntityAttributesToAlertOutbreak,
+} from "../data/repositories/utils/AlertOutbreakMapper";
+import {
+    AlertData,
+    AlertSyncDataStoreRepository,
+} from "../data/repositories/AlertSyncDataStoreRepository";
+
+const RTSL_ZEBRA_DISEASE_TEA_ID = "jLvbkuvPdZ6";
+const RTSL_ZEBRA_HAZARD_TEA_ID = "Dzrw3Tf0ukB";
+const RTSL_ZEBRA_NATIONAL_WATCH_STAFF_USER_GROUP_CODE = "RTSL_ZEBRA_NATONAL_WATCH_STAFF";
+
+function main() {
+    const cmd = command({
+        name: path.basename(__filename),
+        description: "Map national event ID to Zebra Alert Events with no event ID",
+        args: {},
+        handler: async () => {
+            if (!process.env.VITE_DHIS2_BASE_URL)
+                throw new Error("VITE_DHIS2_BASE_URL must be set in the .env file");
+
+            if (!process.env.VITE_DHIS2_AUTH)
+                throw new Error("VITE_DHIS2_AUTH must be set in the .env file");
+
+            const username = process.env.VITE_DHIS2_AUTH.split(":")[0] ?? "";
+            const password = process.env.VITE_DHIS2_AUTH.split(":")[1] ?? "";
+
+            if (username === "" || password === "") {
+                throw new Error("VITE_DHIS2_AUTH must be in the format 'username:password'");
+            }
+
+            const envVars = {
+                url: process.env.VITE_DHIS2_BASE_URL,
+                auth: {
+                    username: username,
+                    password: password,
+                },
+            };
+
+            const api = getD2ApiFromArgs(envVars);
+            const alertRepository = new AlertD2Repository(api);
+            const alertSyncRepository = new AlertSyncDataStoreRepository(api);
+            const notificationRepository = new NotificationD2Repository(api);
+            const optionsRepository = new OptionsD2Repository(api);
+
+            const notifyWatchStaffUseCase = new NotifyWatchStaffUseCase(notificationRepository);
+
+            return Future.joinObj({
+                alertTrackedEntities: alertRepository.getTrackedEntitiesByTEACode({
+                    program: RTSL_ZEBRA_ALERTS_PROGRAM_ID,
+                    orgUnit: RTSL_ZEBRA_ORG_UNIT_ID,
+                    ouMode: "DESCENDANTS",
+                }),
+                hazardTypes: optionsRepository.getAllHazardTypes(),
+                suspectedDiseases: optionsRepository.getAllSuspectedDiseases(),
+            }).run(
```

Comment:
I think you could separate the code into functions with a declarative name to make it easier to understand the steps you are doing in the script